src/core/components/libp2p.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const Node = require('libp2p-ipfs-nodejs')
+const Node = require('../runtime/libp2p-nodejs')
```

Comment:
Perhaps add a comment before explaining that this will be switched to file x in the browser?